### PR TITLE
Update check_api.md

### PR DESCRIPTION
**Pull request by gwaldo**
making the description match the examples

### What does this PR do?

The first param of the metrics is shown as `name`, but the fields description said `metric`.  Alternate suggestion would be to change the param to `metric`

### Motivation

Noticed discrepancy

### Additional Notes

Anything else we should know when reviewing?  Just that I hope you have a good day!

### Describe your test plan

"Write there any instructions and details you may have to test your PR."  Just made sure I didn't mess up the markdown...

### CODE REVIEW

**Comment by olivielpeau** [comment]
Thanks @gwaldo, can you sign the CLA? I'll merge right after

**Comment by olivielpeau** [comment]
Actually, just realized this PR is against `6.2.x` (a very old release branch), that documentation was moved to the documentation website since then. Any reason you were looking at this branch?

**Review by kayayarai — APPROVED** [review]
looks good from a docs perspective

**Pull request closed without merging**
